docs/dependency-updates.md

```diff
@@ -0,0 +1,164 @@
+see code coverage for the frontend.
+
+### `@playwright/test` and `dotenv`
+
+Our Playwright tests are currently only triggered in CI once a day from the
+`main` branch. Thus, there's unfortunately no easy way to verify that it still
+runs successfully until after merging. To do so, after merging the PR, open the
+"Actions" tab in GitHub, then find the "[Relay e2e
+Tests](https://github.com/mozilla/fx-private-relay/actions/workflows/playwright.yml)"
+workflow on the left-hand side, and then use the "Run workflow" trigger button
+in the top row of the table.
+
+`dotenv` is used to load environment variables for Playwright from `.env` files,
+so can be verified in the same way.
+
+### `react-aria` and `react-stately`
+
+These dependencies handle most of the dynamic UI elements in our frontend, such
+as the user menu and the app menu in the header, the modals when claiming a
+subdomain or creating a new custom mask, or the toggle that allows you to set a
+mask to block nothing, promotional emails, or everything. To verfy these
+updates, open the preview deployments and check that you can still interact with
+those using both a keyboard and a mouse.
+
+### `@fluent/*`
+
+[Fluent](https://projectfluent.org/) handles localisation, so verifying these
+involves setting your browser language to something other than English, and
+verifing that the website content indeed shows up in your chosen different
+language.
+
+### `next`
+
+[Next.js](https://nextjs.org/) handles most of our frontend build pipeline and
+bundling the website. To spot-check this update, open the Preview deployment to
+verify that the website was put together successfully, and that you can navigate
+between different pages (e.g. from the home to the FAQ) without issues.
+
+### `@next/bundle-analyzer`
+
+We use this package to investigate if any of our dependencies take up an
+inordinate amount of bytes in our payload. To verify that this still runs
+successfully, run
+
+    ANALYZE=true npm run build
+
+in the `frontend` directory locally, after installing the updated dependency. It
+should create an overview of the client bundle in
+`/frontend/.next/analyze/client.html`, which should look similar to the same
+output in the `main` branch.
+
+### `swr` and `msw`
+
+[SWR](https://swr.vercel.app/) is responsible for making requests to the
+backend, whereas [MSW](https://mswjs.io/) intercepts API calls in the preview
+deployment and returns mock data instead. Both can be verified by opening the
+preview deployment and making sure that you can still interact with the API
+(e.g. by creating a new mask).
+
+### `sass` and `@mozilla-protocol/core`
+
+[Sass](https://sass-lang.com/) gives us some handy tooling to ease working with
+CSS, and [Protocol](https://protocol.mozilla.org/) is Mozilla's design system
+from which we use some Sass variables. Both can be verified by checking the
+preview deployment and making sure the styling doesn't look weird.
```

Comment:
suggestion: You may add an extra line about how Protocol adds "migration tips" to their releases if anything is broken with new updates.

https://github.com/mozilla/protocol/releases/tag/v16.0.0

Reply:
TIL, nice! But I think a number of our dependencies do the same (see e.g. the changelog in https://github.com/mozilla/fx-private-relay/pull/3164 referencing a migration guide), so I think that's covered by this note at the top:

> and a cursory inspection of the changelog (if any) for anything that might be relevant to Relay

So it might be a bit noisy to add that to every dependency that does that?